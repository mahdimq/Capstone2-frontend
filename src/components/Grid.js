import React from 'react';
import PropTypes from 'prop-types';
import { StyledGrid, StyledGridContent } from '../styles/StyledGrid';

function Grid({ header, children }) {
	return (
		<StyledGrid>
			<h1>{header}</h1>
			<StyledGridContent>{children}</StyledGridContent>
		</StyledGrid>
	);
}

// Verify Proptypes
Grid.propTypes = {
	header: PropTypes.string
};
export default Grid;
