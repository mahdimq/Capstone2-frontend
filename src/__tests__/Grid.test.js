import React from 'react';
import { render } from '@testing-library/react';
import Grid from '../components/Grid';

it('renders without crashing', function () {
	render(<Grid />);
});

it('matches snapshot', function () {
	const { asFragment } = render(<Grid />);
	expect(asFragment()).toMatchSnapshot();
});
