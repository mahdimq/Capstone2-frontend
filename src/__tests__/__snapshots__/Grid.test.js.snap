// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-bdfBwQ iKoWzo"
  >
    <h1 />
    <div
      class="sc-gsTCUz iUQxFM"
    />
  </div>
</DocumentFragment>
`;
